common/src/main/java/dev/ftb/mods/ftblibrary/ui/MenuScreenWrapper.java

```diff
@@ -65,9 +63,9 @@ public boolean mouseReleased(double x, double y, int button) {
 	}
 
 	@Override
-	public boolean mouseScrolled(double x, double y, double scroll) {
+	public boolean mouseScrolled(double x, double y, double scroll, double direction) {
```

Comment:
This might actually not be direction, I think it's changed to be scroll x and scroll y so this may have broken some scrolling logic if I'm right